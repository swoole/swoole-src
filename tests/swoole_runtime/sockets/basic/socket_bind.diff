001+ Fatal error: socket_create(): Return value must be of type Socket|false, Swoole\Coroutine\Socket returned in Unknown on line 0
001- bool(true)
002- bool(true)
003- int(16)
004- NULL