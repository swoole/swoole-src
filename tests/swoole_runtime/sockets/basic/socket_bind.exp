bool(true)
bool(true)
int(16)
NULL