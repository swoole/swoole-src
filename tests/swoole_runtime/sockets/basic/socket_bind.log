
---- EXPECTED OUTPUT
bool(true)
bool(true)
int(16)
NULL
---- ACTUAL OUTPUT
Fatal error: socket_create(): Return value must be of type Socket|false, Swoole\Coroutine\Socket returned in Unknown on line 0
---- FAILED
