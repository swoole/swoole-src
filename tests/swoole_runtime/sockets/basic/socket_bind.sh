#!/bin/sh

/home/htf/bin/php   -d "output_handler=" -d "open_basedir=" -d "disable_functions=" -d "output_buffering=Off" -d "error_reporting=32767" -d "display_errors=1" -d "display_startup_errors=1" -d "log_errors=0" -d "html_errors=0" -d "track_errors=0" -d "report_memleaks=1" -d "report_zend_debug=0" -d "docref_root=" -d "docref_ext=.html" -d "error_prepend_string=" -d "error_append_string=" -d "auto_prepend_file=" -d "auto_append_file=" -d "ignore_repeated_errors=0" -d "precision=14" -d "serialize_precision=-1" -d "memory_limit=128M" -d "log_errors_max_len=0" -d "opcache.fast_shutdown=0" -d "opcache.file_update_protection=0" -d "opcache.revalidate_freq=0" -d "opcache.jit_hot_loop=1" -d "opcache.jit_hot_func=1" -d "opcache.jit_hot_return=1" -d "opcache.jit_hot_side_exit=1" -d "zend.assertions=1" -d "zend.exception_ignore_args=0" -d "zend.exception_string_param_max_len=15" -d "short_open_tag=0" -d "session.auto_start=0" -d "zlib.output_compression=Off" -d "mbstring.func_overload=0" -f "/home/htf/workspace/swoole/tests/swoole_runtime/sockets/basic/socket_bind.php"  2>&1
