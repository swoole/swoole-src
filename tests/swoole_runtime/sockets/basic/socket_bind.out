Fatal error: socket_create(): Return value must be of type Socket|false, Swoole\Coroutine\Socket returned in Unknown on line 0